src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts.mdx

```diff
@@ -64,11 +64,14 @@ These are the available notification channel types.
     id="user"
     title="User"
   >
-    For your convenience, we automatically load all users and their email addresses for the selected account. If your account has one or more [sub-accounts](/docs/accounts/install-new-relic/account-setup/manage-apps-or-users-sub-accounts), the notification channel includes only users for the [currently selected master or sub-account](/docs/accounts/install-new-relic/account-setup/use-multiple-accounts).
+    Requirements: 
+      * This feature is not available for users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models), only to users on the original user model. As a workaround, you can use the [email notification channel](#email). 
```

Comment:
I kind of want to set this requirements info out in a tip or caution callout. With our current style, the content sort of bleeds into the rest of the collapser.

Reply:
i'm a bit torn on callouts. in my opinion, i think the callouts are actually more likely to be skipped over as they can seem like supplemental info when scanning. my take, and why i'm not a big fan of them.